.github/CODEOWNERS

```diff
@@ -735,6 +735,8 @@
 /core/domain/role_services*.py @seanlip
 /core/storage/ @seanlip
 /core/domain/user*.py @seanlip
+/extensions/objects/templates/filepath-editor.component*.ts @kevintab95
```

Comment:
Thanks for adding an explanation in the PR description, as per the explanation it doesn't look like that this file needs to be a critical file, can you move these files after `/extensions/objects/`?

Reply:
Done.